### DISCUSSION TITLE
Return the old style for profile pixel icons

### DISCUSSION THREAD

**Discussion by ifsdk** [community/community#28237] [Other Feature Feedback, Questions, & Ideas]
You are making some mistakes I very upset on. Finally I got there to point them out. The one your stupid change there is about pixel icons (random generated pictures at place of avatars). They were cool looking, and I was glad by my generated icon. But some time ago you've changed their style to stupid circled as for avatars (as in stupid monkeys social networks), and now they and my in particular look disappointingly ugly! So in my opinion web code apes (who for example can't even support not so old versions of browsers) should return the old style for profile pixel icons when there are no avatar.

**Comment by Andre601** [comment]
Kind of a rude thing to say... It's from what I can tell only a visual thing and could be easily reverted using some CSS manipulation using an extension.

To prove that the images are unaffected is here your avatar:
![avatar](https://avatars.githubusercontent.com/u/61660977?v=4)

As you can see is it still square. This means that the round avatars you complain about are just a visual thing which can be altered with some CSS magic.
If you like can you suggest this to the [enhanced-github](https://github.com/softvar/enhanced-github) devs to add as a new setting/feature to their extension.

But writing a somewhat rude reply to people who just do their jobs is in no way a nice thing.

**Reply by ifsdk** [reply]
Thx But I know that it is just a css deal and I'm obviously requesting the web server side change, not an extension just to look it only to me. Also as I commented in #18533 it would be yet useful in some cases.
Changing it then was so bad taste, stupid and easy to avoid still providing fancy circled avatars that there is a question what do that people for their money and why it isn't my money for requesting such an obviously thing.
AND I do still need to go somewhere else and SUGGEST yet another time?!

**Reply by jfullstackdev** [reply]
we need to consider all users, so I think it's a general update
